docs/Metalnx-Architecture.md

```diff
@@ -184,7 +164,7 @@ Helper class to retrieve the user currently logged into the application.
 Very useful when it's necessary to do something before or after each request. The `HttpResponseHandlerInterceptor` will intercept all HTTP responses from the server to the client and add few params: user details, metalnx version and URL mapping.
 
 #### modelattribute
-This package contains all representation to objects that will go to the frontend. BreadCrumb, Collection forms, Template forms, Collections forms, URLs, and preferences are all things used on the UI to properly render the page.
+This package contains all representation to objects that will go to the frontend. Collections and URLs are things used on the UI to properly render the page.
```

Comment:
Are "collection forms" different from just "collections"? Sorry if that's a dumb question...

Reply:
@JustinKyleJames Didn't see a response to this question?

Reply:
We talked this over and determined that it is unclear what is meant by these couple of sentences. Thus, the old wording is mostly restored save for the duplicate "collection/collections forms" and "preferences" (which have been removed).